activesupport/lib/active_support/concurrency/load_interlock_aware_monitor.rb

```diff
@@ -6,12 +6,24 @@ module ActiveSupport
   module Concurrency
     # A monitor that will permit dependency loading while blocked waiting for
     # the lock.
-    class LoadInterlockAwareMonitor < Monitor
+    class LoadInterlockAwareMonitor < Monitor # :nodoc:
```

Comment:
?

Reply:
Oh, sorry I misunderstood this is a private API. I will fix to keep this public.

Reply:
It's not "major" public API, but it's a useful tool for external users who need to lock inside a possibly-reloading Rails process -- and the call signatures aren't likely to change, so it seems fine to me for it to be exposed.